.github/workflows/release-notifier.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Issue On Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Issue
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.DOCS_PAT }}  # Assuming 'PAT' is your Personal Access Token stored as a secret
+        script: |
+          const issueTitle = `A new release ${context.payload.release.tag_name} just cut for ${context.payload.repository.name}.`;
+          const issueBody = `Please check the api docs for compatibility with the release. Here is the [link to release notes](${context.payload.release.html_url}).`;
```

Comment:
Is `api docs` correct? Could we articulate in more detail?

Reply:
it is really just a signal for devrel so doesn't matter too much what it says (just api docs are most serious - not too much an issue for ssi but has been for web5). It is the same signal the use for that to ensure things are ok.